wireless_modules/wireless_module.py

```diff
@@ -2,22 +2,26 @@
 import machine
 import ubinascii
 import ujson
+import sys
 import time
 
 from mqtt_client import Client
+from status_led import WmState
 
 try:
     import config
 except FileNotFoundError:
     print("Error importing config.py, ensure a local version of config.py exists")
 
+LOW_BATTERY_THRESHOLD = 3.3
```

Comment:
It's probably not a concern for the short-medium term, but is it possible in the future for different WM to use different types of batteries and hence have different low battery thresholds?

Reply:
Very very unlikely. All lithium chemistry batteries have a 3.7 volt nominal voltage and it'd be fairly safe to say that 3.3 will always be a good threshold (or if we decide it's too low or too high, we'd want to adjust it for all WMs). Only possibility of a different voltage is if we use multiple cells, but currently we're looking at significantly reduced power consumption, not increased. I think this is fine.